src/bundle/Resources/public/js/scripts/admin.content.draft.edit.js

```diff
@@ -0,0 +1,25 @@
+(function (global, doc) {
+    const editVersion = (event) => {
+        const contentId = event.currentTarget.dataset.contentId;
+        const versionNo = event.currentTarget.dataset.versionNo;
+        const languageCode = event.currentTarget.dataset.languageCode;
+        const contentDraftEditUrl = event.currentTarget.dataset.href;
+        const checkLink = window.Routing.generate('ezplatform.version.check', { contentId, versionNo, languageCode });
+
+        event.preventDefault();
+
+        let request = $.get(checkLink);
+
+        request.fail(function(xhr) {
+            if(xhr.status === 409){
+                $('.modal-content').html(xhr.responseText);
+                $('#draft-has-conflict-modal').modal('show');
+            }
+            if(xhr.status === 301){
```

Comment:
I discussed it with @dew326 and it's kind of awkward that the endpoint won't ever return 2xx status. Also from the endpoint's POV 301 would mean that the endpoint is moved permanently which is not true. I'd opt for `204 No Content` status here just to confirm that there is no version conflict.

Also there are code style infringements.